packages/ui/components/form/inputs/Input.tsx

```diff
@@ -255,7 +256,8 @@ type TextAreaProps = JSX.IntrinsicElements["textarea"];
 
 export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(function TextAreaInput(props, ref) {
   return (
-    <textarea
+    <TextareaAutosize
+      // @ts-expect-error how does typescript work? halp.
```

Comment:
This is the only thing I'm not sure how to fix yet. If anyone has any suggestions, otherwise, I'll keep on it.